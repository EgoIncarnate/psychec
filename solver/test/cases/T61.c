// example 1
m() {} // gen alpha = int

// example 2
void h() { /* no ret */ } // alpha = void

// example 3
int f() { /* no ret */ } // alpha = int

// example 4
Foo g() { /* no ret */ } // alpha = Foo and foo is orphan => uintptr_t = Foo

T3 a() {}
void b() { T3 x = 9.9; }