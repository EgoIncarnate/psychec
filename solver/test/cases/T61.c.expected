typedef double T3;
typedef int /*<<< orphan or scalar*/ Foo;
